src/topology/algebra/ordered.lean

```diff
@@ -2536,6 +2536,11 @@ lemma intermediate_value_Icc' {a b : α} (hab : a ≤ b) {f : α → δ} (hf : c
   Icc (f b) (f a) ⊆ f '' (Icc a b) :=
 is_preconnected_Icc.intermediate_value (right_mem_Icc.2 hab) (left_mem_Icc.2 hab) hf
 
+/--Intermediate Value Theorem for continuous functions on closed intervals, unordered case.-/
```

Comment:
```suggestion
/-- Intermediate Value Theorem for continuous functions on closed intervals, unordered case. -/
```
I feel ridiculous pointing this out, sorry. :-)

Reply:
Now worries! I suppose I didn't notice it myself since the docstrings for the previous two lemmas don't have that whitespace.